etc/ipxe/dracut.ipxe

```diff
@@ -0,0 +1,7 @@
+#!ipxe
+
+set uri http://{{.Ipaddr}}:{{.Port}}/provision/{{.Hwaddr}}?assetkey=${asset}&uuid=${uuid}
+
+kernel --name kernel ${uri}&stage=kernel
+initrd --name initramfs tftp://192.168.3.11/warewulf/initramfs.img
```

Comment:
if we plan to use the code
https://github.com/warewulf/warewulf/pull/1186/files#diff-70e45b6fb2d31ccdf8709f7d50e3539534e46b19c01ff487e70f9cf880b1763eR218-R224
we can use `initrd --name initramfs ${uri}?stage=initramfs`